data/secscan_model/secscan_v4_model.py

```diff
@@ -163,6 +170,10 @@ def eligible_manifests(base_query):
         if max_id is None or min_id is None or min_id > max_id:
             return None
 
+        reindex_threshold = lambda: datetime.utcnow() - timedelta(
```

Comment:
Let's add a test for this: write the MSS row, change the indexer state, and make sure it isn't picked up again since we're within the reindexing exclusion window

Reply:
I already did [here](https://github.com/quay/quay/pull/373/files#diff-d1556d15ee5435b7ee2ca49b3a797803R279).